<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCoolPlayerClass</class>
 <widget class="QMainWindow" name="QCoolPlayerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCoolPlayer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionExit_0"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuStatus">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Ctrl</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionRealTime"/>
    <addaction name="separator"/>
    <addaction name="actionFullScreen_F11"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actiontake_picture"/>
    <addaction name="actionrecord_video"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStatus"/>
   <addaction name="menuCamera"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenFile_H264">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OpenFile(H264)</string>
   </property>
  </action>
  <action name="actionExit_0">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="action25Fps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>25Fps</string>
   </property>
  </action>
  <action name="action30Fps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>30Fps</string>
   </property>
  </action>
  <action name="action60Fps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>60Fps</string>
   </property>
  </action>
  <action name="action24Fps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>24Fps</string>
   </property>
  </action>
  <action name="action50Fps">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50Fps</string>
   </property>
  </action>
  <action name="actionSkyMode">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>SkyMode</string>
   </property>
  </action>
  <action name="actionGroundMode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GroundMode</string>
   </property>
  </action>
  <action name="actionFullScreen_F11">
   <property name="text">
    <string>FullScreen（F11）</string>
   </property>
  </action>
  <action name="actionConnectGround">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ConnectGround</string>
   </property>
  </action>
  <action name="actionRealTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Realtime</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actiontake_picture">
   <property name="text">
    <string>Take Picture</string>
   </property>
  </action>
  <action name="actionrecord_video">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Record Video</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qcoolplayer.qrc"/>
 </resources>
 <connections/>
</ui>
